app/views/pages/home.html.erb

```diff
@@ -59,3 +59,18 @@
         </div>
       </div>
     </div>
+
+
+<% content_for(:after_js) do %>
+ <script>
```

Comment:
@Spone yes, this option to put it in the assets and use it everywhere did not work somehow. I rather add it once and use it everywhere...